src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -103,7 +121,14 @@ export default {
             const lastHistoryTime = this.timeFormat.format(this.imageHistory.slice(-1)[0]);
             const lastHistoryURL = this.imageFormat.format(this.imageHistory.slice(-1)[0]);
 
-            return (datumTime === lastHistoryTime) && (datumURL === lastHistoryURL);
+            // datum is not valid if it matches the last datum in history,
+            // or it is before the last datum in the history
+            const datumTimeCheck = datum[this.timeKey];
+            const historyTimeCheck = this.imageHistory.slice(-1)[0][this.timeKey];
```

Comment:
Run it through `this.timeSystem.parse()`.

Reply:
Updated